README.md

```diff
@@ -9,7 +9,7 @@
 yarn add --dev jest-styled-components
```

Comment:
If you don't mind I would ask you to revert this. 
I'll be happy to update the README once it's merged.